app/routers/answer.py

```
@@ -0,0 +1,29 @@
+@router.post("/{quiz_id}/answer", status_code=status.HTTP_201_CREATED)
+async def save_quiz_answers(
+    quiz_id: int, answer_data: AnswerSheetCreate, db: AsyncSession = Depends(get_db)
+):
+    answer_service = AnswerService(db)
+    return await answer_service.save_answers(quiz_id=quiz_id, answer_data=answer_data)
+
+
+@router.get("/answers/{answersheet_id}", response_model=AnswerSheetResponse)
+async def get_answer_sheet(
+    answersheet_id: int,
+    db: AsyncSession = Depends(get_db),
```

Ruff (0.8.2)

Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)